backend/typescript/middlewares/validators/camperValidators.ts

```diff
@@ -9,123 +9,148 @@ import {
 
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 /* eslint-disable-next-line import/prefer-default-export */
-export const createCamperDtoValidator = async (
+export const createCampersDtoValidator = async (
   req: Request,
   res: Response,
   next: NextFunction,
 ) => {
-  if (!validatePrimitive(req.body.campSession, "string")) {
-    return res.status(400).send(getApiValidationError("campSession", "string"));
-  }
-  if (!validatePrimitive(req.body.firstName, "string")) {
-    return res.status(400).send(getApiValidationError("firstName", "string"));
-  }
-  if (!validatePrimitive(req.body.lastName, "string")) {
-    return res.status(400).send(getApiValidationError("lastName", "string"));
-  }
-  if (!validatePrimitive(req.body.age, "integer")) {
-    return res.status(400).send(getApiValidationError("age", "integer"));
-  }
-  if (req.body.allergies && !validatePrimitive(req.body.allergies, "string")) {
-    return res.status(400).send(getApiValidationError("allergies", "string"));
-  }
-  if (req.body.hasCamera && !validatePrimitive(req.body.hasCamera, "boolean")) {
-    return res.status(400).send(getApiValidationError("hasCamera", "boolean"));
-  }
-  if (req.body.hasLaptop && !validatePrimitive(req.body.hasLaptop, "boolean")) {
-    return res.status(400).send(getApiValidationError("hasLaptop", "boolean"));
-  }
-  if (
-    req.body.earlyDropoff &&
-    (!validatePrimitive(req.body.earlyDropoff, "string") ||
-      !validateTime(req.body.earlyDropoff))
-  ) {
+  if (req.body.length === 0) {
     return res
       .status(400)
-      .send(getApiValidationError("earlyDropoff", "24 hr time string"));
+      .send(
+        "No campers sent - there must be at least one camper in the request.",
+      );
   }
-  if (
-    req.body.latePickup &&
-    (!validatePrimitive(req.body.latePickup, "string") ||
-      !validateTime(req.body.latePickup))
-  ) {
-    return res
-      .status(400)
-      .send(getApiValidationError("latePickup", "24 hr time string"));
-  }
-  if (
-    req.body.specialNeeds &&
-    !validatePrimitive(req.body.specialNeeds, "string")
-  ) {
-    return res
-      .status(400)
-      .send(getApiValidationError("specialNeeds", "string"));
-  }
-  if (!Array.isArray(req.body.contacts) || req.body.contacts.length !== 2) {
-    return res
-      .status(400)
-      .send("There must be 2 emergency contacts specified.");
-  }
-  for (let i = 0; i < req.body.contacts.length; i += 1) {
-    const contact = req.body.contacts[i];
-    if (!validatePrimitive(contact.firstName, "string")) {
+  const { campSession } = req.body[0];
+  const { chargeId } = req.body[0];
```

Comment:
nit: we can define them together
```suggestion
  const { campSession, chargeId } = req.body[0];
```